spec/unit/yardstick/class_methods/measure_spec.rb

```diff
@@ -8,10 +8,10 @@
 
     it 'delegates to Processor' do
       processor = double('processor')
-      Yardstick::Processor.stub(:new).with(instance_of(Yardstick::Config)) do
+      allow(Yardstick::Processor).to receive(:new).with(instance_of(Yardstick::Config)) do
```

Comment:
Instead of using the block to define the return value, I think that `and_return(processor)` would use a less powerful primitive.